static/app/views/dashboards/detail.tsx

```diff
@@ -1245,7 +1245,7 @@ class DashboardDetail extends Component<Props, State> {
                                     onEditWidget={this.onEditWidget}
                                   />
 
-                                  <DevWidgetBuilder
+                                  <WidgetBuilderV2
```

Comment:
Technically v3, but I'll leave it for now lol

Reply:
lolll